ouroboros-praos/src/consensus/mod.rs

```diff
@@ -0,0 +1,349 @@
+use malachite::platform_64::Limb;
+use malachite::Natural;
```

Comment:
Ideally, we shouldn't depend on those directly but solely rely on pallas_math? Let's perhaps leave some explicit `TODO` marker here to remember getting rid of the dependencies.

Reply:
I had already done this in pallas-math. Unfortunately, I didn't use it..... fixed.